tests/paramgrill.c

```diff
@@ -491,7 +1187,7 @@ static void BMK_benchFullTable(ZSTD_CCtx* cctx, ZSTD_DCtx* dctx, const void* src
     winnerInfo_t winners[NB_LEVELS_TRACKED+1];
     const char* const rfName = "grillResults.txt";
     FILE* const f = fopen(rfName, "w");
-    const size_t blockSize = g_blockSize ? g_blockSize : srcSize;   /* cut by block or not ? */
+    const size_t blockSize = g_blockSize ? g_blockSize : ZSTD_BLOCKSIZE_MAX;   /* cut by block or not ? */
```

Comment:
Any reason you prefer to change the default value to `ZSTD_BLOCKSIZE_MAX` ?
This will directly impact the selection of `l1params` below, and I'm not sure to understand why it's preferable.